tests/unit/api/objectPut.js

```diff
@@ -201,6 +201,10 @@ describe('objectPut API', () => {
             });
     });
 
+    /**
+     * now overwriting an object in versioning means adding a new version, thus
+     * old data will node be deleted just be overwriting an existing object
```

Comment:
now overwriting an object in versioning means adding a new version, thus old data will not be deleted